src/components/common/PeopleAndRoles.vue

```diff
@@ -189,17 +188,8 @@ export default class PeopleAndRoles extends Vue {
     // first assign empty org/person object
     this.currentOrgPerson = cloneDeep(EmptyOrgPerson)
 
-    if (roleType) {
-      // a role was provided - pre-select it
-      this.currentOrgPerson.roles = [{ roleType }]
-    } else if (this.validNumCompletingParty && !this.getPeopleAndRolesResource.addIncorporator) {
-      // only Director role is possible - pre-select it
```

Comment:
We can no longer assume that this must be a Director if we already have the completing party and we don't need to add incorporator(s). There are too many other things to check and it's too complicated to generalize this.

Actually, I think "feature" was broken early on (probably when we allowed the replacement of the completing party) because I can't find any examples of adding a person whose only role can be Director.